internal/commands/verify.go

```diff
@@ -78,14 +78,22 @@ func NewVerifyCommand(ctx context.Context) *cobra.Command {
 				return fmt.Errorf("unmarshal parameters: %w", err)
 			}
 
-			results, err := runner.Run(ctx)
+			results, raw, err := runner.Run(ctx)
 			if err != nil {
 				return fmt.Errorf("running verification: %w", err)
 			}
 
 			outputter := output.Get(runner.Output, output.Options{NoColor: runner.NoColor, Tracing: runner.Trace, ShowSkipped: true})
-			if err := outputter.Output(results); err != nil {
-				return fmt.Errorf("output results: %w", err)
+
+			// report currently available with stdout only
+			if runner.Report && runner.Output == output.OutputStandard {
```

Comment:
As this is, it silently fails to create a report if the user specifies a different output format. I think it would be a better user experience if we check that the output is not supported with the report flag and then surface that error to the user. This also allows us to return earlier, before running any of the tests and removes the need for this nested if/else statement.

Reply:
agree. fixed.